src/redux/account/actionCreator.js

```diff
@@ -0,0 +1,53 @@
+import { Alert } from 'react-native';
+import { BACKEND_URL } from 'react-native-dotenv';
+import {
+  LOGIN_USER_START,
+  LOGIN_USER_FAILURE,
+  LOGIN_USER_SUCCESS,
+  LOGOUT_SUCCESS,
+} from './actionType';
+
+const loginUserStart = () => ({
+  type: LOGIN_USER_START,
+});
+
+const loginUserSuccess = (payload) => ({
+  type: LOGIN_USER_SUCCESS,
+  payload,
+});
+
+export const loginUserFail = (payload) => ({
+  type: LOGIN_USER_FAILURE,
+  payload,
+});
+
+const loginUser = (username, password) => (dispatch) => {
+  const loginURL = `${BACKEND_URL}/users/login`;
+  fetch(loginUserStart);
+  fetch(loginURL, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      username,
+      password,
+    }),
+  })
+    .then((response) => {
+      if (response.status === 200) {
+        return response.json();
+      } if (response.status === 401) {
+        Alert.alert('Your username or password is incorrect');
```

Comment:
do not do any UI stuff in the action, lets dipatch en error and handle that in the corresponding component